Find a functionally equivalent code:android.widget.CalendarView.getSelectedDateVerticalBar:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the drawable for the vertical bar shown at the beginning and at
 * the end of the selected date.
 *
 * @return The vertical bar drawable.
 */

@@ -1,3 +1,3 @@
 {
-    return mSelectedDateVerticalBar;
+    return mDelegate.getSelectedDateVerticalBar();
 }

